Examples/Io/Root/src/RootAthenaDumpReader.cpp

```diff
@@ -239,9 +225,12 @@ RootAthenaDumpReader::RootAthenaDumpReader(
                                  &DTTstTrack_subDetType);
   m_inputchain->SetBranchAddress("DTTstCommon_subDetType",
                                  &DTTstCommon_subDetType);
+  */
 
-  m_inputchain->Add(m_cfg.inputfile.c_str());
-  ACTS_DEBUG("Adding file " << m_cfg.inputfile << " to tree" << m_cfg.treename);
+  for (const auto& file : m_cfg.inputfiles) {
+    m_inputchain->Add(file.c_str());
+    ACTS_DEBUG("Adding file '" << file << "' to tree " << m_cfg.treename);
```

Comment:
nitpick: in case of an error this might be better

```suggestion
    ACTS_DEBUG("Adding file '" << file << "' to tree " << m_cfg.treename);
    m_inputchain->Add(file.c_str());
```